robottelo/ui/base.py

```diff
@@ -237,6 +237,28 @@ def wait_until_element_is_clickable(
             )
             return None
 
+    def wait_until_element_is_not_visible(
+            self, locator, timeout=12, poll_frequency=0.5):
+        """Wrapper around Selenium's WebDriver that allows you to pause your
+        test until an element in the web page disappeared and is not present
+        and is not visible anymore.
+
+        """
```

Comment:
What does this method do? It waits until a web page element is gone. How does it do it? It wraps Selenium WebDriver. Can you re-write this docstring so that it has a summary (the "what") followed by details? Something like this:

> Wait until a web page element is not present.
>
> :param locator: ???
> :param timeout: How long should this method wait for the element to disappear?
> :param poll_frequency: How frequently should this method check for the presence of the element?
> :returns: If the page element does not return by the ``timeout``, return ``None``. Otherwise, return the web page element.